pyuploadcare/api_resources.py

```
@@ -65,6 +66,20 @@ def __str__(self):
         return self.cdn_url
 
     @property
+    def uuid(self):
+        return self._uuid
+
+    @uuid.setter
+    def uuid(self, value):
+        match = RE_UUID_REGEX.match(value)
```

this will silently eat invalid values